a.rb

```diff
@@ -0,0 +1,5 @@
+p "hoge"
+
+def foo val
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Unused method argument - val. If it's necessary, use _ or _val as an argument name to indicate that it won't be used. You can also write as foo(*) if you want the method to accept any arguments but don't care about them.